src/main/resources/aws-smtp-relay.service

```diff
@@ -0,0 +1,14 @@
+# /usr/lib/systemd/system/tor.service
+[Unit]
+Description=AWS SMTP Relay
+After=network.target
+
+[Service]
```

Comment:
How about `SuccessExitStatus=130 143`? I believe Java will exit with one of those codes on success and this is needed so the unit doesn't enter a "failed" state

Reply:
Agreed, fixed